app/models/criteria.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
 # rubocop:disable Rails/FindEach
 class Criteria
-  ATTRIBUTES = CriteriaHash.reduce([]) do |attributes, criterion|
-    attributes | criterion[1].keys
-  end.map(&:to_sym).freeze
-  FUTURE_ATTRIBUTES = %i(
-    met_url na_placeholder na_suppress unmet
-    unmet_suppress
+  ATTRIBUTES = %i(
+    autofill category description details future major met_placeholder
+    met_suppress met_url_required minor na_allowed na_justification_required
+    na_placeholder rationale unmet_placeholder
   ).freeze
+  FUTURE_ATTRIBUTES = %i(met_url na_suppress unmet unmet_suppress).freeze
```

Comment:
I don't think there's any reason to have "FUTURE_ATTRIBUTES"; just have a single list of ATTRIBUTES.

Also, I think it might be better if they were organized more clearly. I suggest using the same order as the leading comments in criteria.yml (and if they're confusing, make them match).  Something like this:

 ATTRIBUTES = %i(
  category future
  na_allowed na_justification_required met_url_required
  description details
  met_placeholder unmet_placeholder na_placeholder
  met_suppress unmet_suppress
  rationale autofill
  major minor).freeze

I think we need "major" and "minor", so that we can query them later (minor was omitted) - let me know if that's incorrect.